### ISSUE TITLE
last entry in "class_name" is used everywhere to generate new instance

### ISSUE THREAD

**Issue by stevleibelt** [bazzline/php_component_locator_generator#5]
example generated locator

```php
//....

    /**
     * @return \Net\Bazzline\Component\Cli\Arguments\Arguments
     */
    public function getCliArguments()
    {
        return $this->fetchFromSharedInstancePool('\Net\Bazzline\Component\CommandCollection\Filesystem\Remove');
    }

    /**
     * @return \Net\Bazzline\Component\ApiDocumentBuilder\Builder\Apigen
     */
    public function getApigenBuilder()
    {
        return $this->fetchFromSharedInstancePool('\Net\Bazzline\Component\CommandCollection\Filesystem\Remove');
    }
//...
```

**Comment by stevleibelt** [comment]
Quickfix would be to adapt "\Net\Bazzline\Component\Locator\Configuration\Instance".

```php
    public function getMethodBodyBuilder()
    {
        return $this->methodBodyBuilder;
    }
```

to

```php
    public function getMethodBodyBuilder()
    {
        $this->methodBodyBuilder->setInstance($this);

        return $this->methodBodyBuilder;
    }
```

**Comment by stevleibelt** [comment]
fixed with commit [14ba83e128f0621928f368e6137eb808ffdb989c](https://github.com/bazzline/php_component_locator_generator/commit/14ba83e128f0621928f368e6137eb808ffdb989c)

**Comment by stevleibelt** [comment]
Fixed in release [2.0.0](https://github.com/bazzline/php_component_locator_generator/tree/2.0.0)
